### ISSUE TITLE
restore the model without tensorpack

### ISSUE THREAD

**Issue by zuokai** [ppwwyyxx/tensorpack#534]
I use tensorpack train the model and save a tensorflow model，then I want to restore the model without tensorpack, the code is as follows:
   with tf.name_scope('train'):
        loss = custom_function.loss_function(y, y_)
        train_op = custom_function.train_function(loss, learn_rate)
        accuracy_op = custom_function.accuracy_function(y, y_)
        output_result = custom_function.output_result(y)
        init_op = tf.group(tf.global_variables_initializer(), tf.local_variables_initializer())
        saver = tf.train.Saver()
        sess_config = tf.ConfigProto(allow_soft_placement=True, log_device_placement=False)
        sess_config.gpu_options.allow_growth = True
        sess = tf.Session(config=sess_config)
        merged_summary = tf.summary.merge_all()
        train_writer = tf.summary.FileWriter(log_dir, sess.graph)
        #saver = tf.train.import_meta_graph('logs/model_5.meta')
        sess.run(init_op)
        coord = tf.train.Coordinator()
        threads = tf.train.start_queue_runners(sess=sess, coord=coord)
        saver.restore(sess, "model_meishi/model-6690")
but I can not restore successfully，when I run the code，there is no error， but the result is not right，what‘s wrong with my code？

**Comment by ppwwyyxx** [comment]
Not a tensorpack issue. As the model is just in standard checkpoint format and you can definitely load it.

There could be millions of reasons why the "result is not right". You have to confirm it's because of model restoring problem. I would `sess.run()` a weight tensor after restoring, and see if it equals the one saved in the checkpoint.

**Comment by zuokai** [comment]
I just change the batch_size form 1 to 128, everything is ok. but I don‘t know why...

**Comment by ppwwyyxx** [comment]
Is there any tensorpack code involved in your code?..
I don't know what you mean by "result not right". I don't even know what your code is trying to do. From the information you provided it seems like model restoring itself is working fine.

**Comment by ppwwyyxx** [comment]
No indication that this issue is a tensorpack problem. Closing now. Reopen if you have more information to provide.
